float32 xMin
float32 xMax
float32 stepSize
float32[] upBorderline
float32[] downBorderline